tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -50,9 +50,21 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref');
     zypper_call("in -t pattern ${hypervisor}_server ${hypervisor}_tools", 1800);
     set_serial_console_on_vh('', '', $hypervisor);
+    systemctl 'enable libvirtd', ignore_failure => 1;
 
     virt_autotest::utils::install_default_packages();
     save_screenshot;
+
+    #set up necessary network bridge br0 for virtualization tests.
+    #SLE sets up br0 during installation as kvm/xen role, while openSUSE TW does not.
+    #change MACAddressPolicy to none, or the network bridge will get a different MAC address from its enslave
+    #it is specific to TW because it uses udev naming scheme v247, see bsc#1136600
+    my $bridge_rule_file = "/usr/lib/systemd/network/98-default-bridge.link";
+    assert_script_run("echo \"[Match]\nDriver=bridge\n\n[Link]\nMACAddressPolicy=none\" > $bridge_rule_file", 60);
```

Comment:
Be cautious that you overwrite $bridge_rule_file

Reply:
98-default-bridge.link is created as a new file. The existing one is 99-default.link. 98-default-bridge.link is loaded ahead to override the rules for 'bridges', it is deliberate as we'd like the MAC of the bridge is consistent to its enslave network device the way in SLE.

Reply:
What a new rule file like 98-default-bridge.link is needed ? You can just modify the existing one ?

Reply:
I just follow the recommendation in https://bugzilla.opensuse.org/show_bug.cgi?id=1136600#c34. It is safer and easier.

Reply:
So this module switch_to_ssh_and_install_hypervisor.pm is only for TW to use ?

Reply:
Yes, it is specific to TW only as there is no kvm/xen role selection during installation.